knowledge-base/drawer-add-expand-collapse-handle.md

```diff
@@ -102,3 +102,7 @@ To achieve the desired result you can try the following:
     }
 }
 ````
+
+## See Also
+
+* [CSS Isolation does not work for Telerik components]({%slug common-kb-css-isolation%})
```

Comment:
You can also link https://docs.telerik.com/blazor-ui/styling-and-themes/override-theme-styles